com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,169 @@
+app-id: com.warlordsoftwares.formatlab
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: formatlab
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
With host below are redundant, why do you need host?

Reply:
removed host. thanks